tools/Makefile

```diff
@@ -13,5 +13,11 @@ MKIMAGE_INCS := -I../external/lib/mincrypt/include -I../lib/bootimage/include
 mkimage: $(MKIMAGE_SRCS) $(MKIMAGE_DEPS)
 	gcc -Wall -g -o $@ $(MKIMAGE_INCS) $(MKIMAGE_SRCS)
 
+NDEBUG_DEPS :=
+NDEBUG_SRCS := ndebug/ndebug.cc ndebug/usbionode.cc ndebug/tcpionode.cc
+NDEBUG_INCS := -I./ndebug/include -I../lib/ndebug/include
+ndbg: $(NDEBUG_SRCS) $(NDEBUG_DEPS)
+	g++ -std=c++11 -Wall -g -o $@ $(NDEBUG_INCS) $(NDEBUG_SRCS) -L/usr/lib/x86_64-linux-gnu -lusb-1.0
```

Comment:
do you actually need the -L/usr/lib/... path? seems like that would be implicit, and it keeps it from building on mac.